### PR TITLE
update ExaminePlugin to account for ground items

### LINKED ISSUE #6201: Examining ground items does not trigger the Examine plugin

### PR DESCRIPTION
**Pull request by Brodan**
Fixes #6201.

This PR updates a conditional in the `ExaminePlugin` so that price look-ups are triggered when examining items on the ground and a message is posted to chat, the same way it behaves when examining an inventory item.

The diff looks a lot bigger than it is because of indentation changes.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java [hunk 1]
```diff
@@ -252,67 +256,48 @@ public void onChatMessage(ChatMessage event)
 		int widgetChild = TO_CHILD(widgetId);
 		Widget widget = client.getWidget(widgetGroup, widgetChild);
 
-		if (widget == null)
-		{
-			return null;
-		}
+		if (widget != null) {
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java]: Why did you change the guard clause to a massive `if` block and un-do the formatting that was present here? The code here already conformed to our code style guidelines and should not be changed on a whim.

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java]: Revisiting this comment [from the original form of this PR](https://github.com/runelite/runelite/pull/6284/files#r229890170), `findItemFromWidget()` has only one entry point, and it is not reached when examining a ground item.

Comment (Brodan) [reply runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java]: Yea I was referencing my old PR so I could re-open this since so much has changed but it looks like all that needed to be added now is that new examine `case`. I'm trying to figure out what all this code is doing again, but this seems to be working locally just fine.

Comment (Brodan) [reply runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java]: Wow, yea, I think my old PR was just trying waaaay too hard to needlessly refactor code in addition to adding that feature. All this feature really needs is that extra case. Lemme know your thoughts!
